Assets/HoloToolkit/Input/Prefabs/FocusedObjectKeywordManager.prefab

```diff
@@ -0,0 +1,103 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1001 &100100000
```

Comment:
Could you please update the ReadMe.md on what this prefab is for?
Right now I'm not sure what this does.